.github/workflows/pr-manager.yml

```diff
@@ -0,0 +1,21 @@
+name: "Pull Request Manager"
+on:
+  schedule:
+    - cron: '0 * * * *'
+
+jobs:
+  stale:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Process stale issues and PRs
+        uses: Sonia-corporation/stale@2.5.0
+        with:
+          issue-processing: false
+          pull-request-days-before-stale: 14
+          pull-request-days-before-close: 365
```

Comment:
What do you think about making this one smaller? maybe 6 months?

Reply:
Sounds good to me